how-to-pitch-post-supply/readme.md

```diff
@@ -0,0 +1,49 @@
+> Goal of this guide: Help you make it clear what Post.supply is and what it offers
+
+# Your awesomeness is enough. You don't have to be a selling super hero
+<img src="http://66.media.tumblr.com/da6ec2c2cb443e8e7d1ade9eb220c86d/tumblr_nvls4ybWUo1sxfvy5o1_400.gif" width="500px">
+
+First off it is really important that we should never try to sell our product.
+
+We offer a solution to a problem. As long as we are able to explain how we solve the problem, those that have the problem will see the value in what we do.
+
+The product should sell itself, as long as people know what the product is. If not, we need to redefine the product, not try to sell harder.
+
+## The basics of what we do and why we do it.
+
+
+### Problem
+
+Social media is mostly used to influence the way people perceive the sender, creating large amounts of aggressive and pushy content that has no real value for the receiver. This is a waist of resources and drowns out the content that is actually valuable for the receiver.
+
+### Solution
+
+We are convinced that you can have effective communication by using it as a tool to achieve the goals of your company.
```

Comment:
I don't think people "have effective communication", but rather people communicate effectively. and use that communication as a tool.